tutorials/index.md

```diff
@@ -4,9 +4,9 @@ These tutorials will help you get familiar with Chainblocks.
 
 ## Setting up the workspace
 
-We will be using <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a>, although any text editor could work.
+We will be using [`Visual Studio Code`](https://code.visualstudio.com/) although any text editor could work.
```

Comment:
Usually the ` `` ` syntax is used for keywords or special words from source code. For normal links, we don't use them.

Reply:
Alright, thanks. Noted for the future.